app/controllers/support_interface/references_controller.rb

```diff
@@ -15,7 +15,7 @@ def confirm_cancel
     def reinstate; end
 
     def confirm_reinstate
-      RequestReference.new.call(@reference)
+      @reference.update!(feedback_status: 'feedback_requested', audit_comment: 'Reversing reference refusal using Support interface')
       flash[:success] = 'Reference was reinstated'
```

Comment:
Maybe this could read "The referee is able to provide a reference again" or sth?